archinstall/lib/user_interaction.py

```diff
@@ -98,22 +98,43 @@ def sig_handler(sig :int, frame :Any) -> None:
 
 	return True
 
+def is_strong_password(passwd :str) -> bool:
+
+	if not (
+		any(char.isdigit() for char in passwd)
```

Comment:
I use `black` and it suggested that I break the line. I will solve the same now.